### PR TITLE
Update installation-laravel.md

### PR DESCRIPTION
**Pull request by erlangp**

### CODE REVIEW

**Comment by erlangp** [comment]
we must generate & configure the config first, because when i try to run migration:

```
php artisan migrate
Migrating: 2019_12_01_205310_create_permission_tables

   Illuminate\Database\QueryException  : SQLSTATE[42000]: Syntax error or access violation: 1103 Incorrect table name '' (SQL: create table `` (`id` bigint unsigned
```

File: docs/installation-laravel.md [hunk 1]
```diff
@@ -166,7 +166,13 @@ return [
 ];
 ```
 
-After the config & migration have been published you can create the role- and permission-tables by running the migrations:
+Then, apply the configuration:
+
+```bash
+php artisan config:cache
```
Comment (drbyte) [inline docs/installation-laravel.md]: Why run config:cache here?

Comment (erlangp) [reply docs/installation-laravel.md]: without run that command, i got SQL error like i mention before

**Comment by erlangp** [comment]
step i have tried:
1. i publish migration
2. i run migration migrate
3. i got SQL error

4. i  publish config
5. i run migration migate
6. i got SQL error

7. i run config:cache
8. i run migration migrate
9. i got migration Success

**Comment by erlangp** [comment]
so the happy path:
1. publish migration
2. publish config
3. run config:cache
4. run migration migrate

(this i tried on my local computer, windows 8)

**Comment by drbyte** [comment]
I think you're better to run `config:clear` in dev. There's NO need to run config:cache in dev. That's only for production environments.

But I wouldn't put anything about config caching in this package's docs. Those are dev skill things, not package-specific things.

**Comment by drbyte** [comment]
With this package, as long as you've published the config and the migration (which can be done with vendor:publish and choose this package from the list), then when you run the migration things will be fine.

File: docs/installation-laravel.md (hunk 1 above)
Comment (erlangp) [reply docs/installation-laravel.md]: must i change with `config:clear` sir?

**Comment by erlangp** [comment]
i removed that line sir

**Comment by erlangp** [comment]
thanks sir :D

**Pull request merged**